loader.qml

```diff
@@ -1,57 +1,193 @@
 import QtQuick 2.15
 import QtGraphicalEffects 1.12
 import QtQuick.Layouts 1.2
-import QtQuick.Controls 1.4
+import QtQuick.Controls 2.15
 import QtQuick.Dialogs 1.1
 
 ApplicationWindow {
     id: root
+    visible: true
```

Comment:
Why is the visible: true necessary now ?

Reply:
No, it is not.  Removed.